### PR TITLE
add ore endpoint

### PR DESCRIPTION
**Pull request by Lanikean**

### CODE REVIEW

File: proto/api.proto [hunk 1]
```diff
@@ -725,6 +725,17 @@ service Api {
     };
   }
 
+  rpc PostubmitMineOre(PostSubmitMineOreRequest) returns (PostSubmitMineOreResponse) {
+    option (google.api.http) = {
+      post: "/api/v1/mine-ore"
```
Comment (DimaCantemir) [inline proto/api.proto]: /v2/

File: proto/api.proto [hunk 2]
```diff
@@ -725,6 +725,17 @@ service Api {
...
+      body: "*"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      description: "Submits two transactions in sequence for mining ore";
+      summary: "Submit mine ore transactions";
+    };
```
Comment (DimaCantemir) [inline proto/api.proto:738]: please add a tag -  tags: ["Ore"];
this way it will be grouped separately in the swagger doc

File: proto/api.proto [hunk 3]
```diff
@@ -840,6 +851,15 @@ enum MarketStatus {
   MS_ONLINE = 1; // online
 }
 
+message PostSubmitMineOreResponse {
+  string status = 1;
```
Comment (DimaCantemir) [inline proto/api.proto]: what's this for ? not sure we need a response since the miner can't do anything with it anyway

File: proto/api.proto [hunk 4]
```diff
@@ -840,6 +851,15 @@ enum MarketStatus {
...
+}
+
+message PostSubmitMineOreRequest {
+  PostSubmitRequest first_transaction = 1;
```
Comment (DimaCantemir) [inline proto/api.proto]: no need to have 2 dedicated fields with complex format...just do:
` repeated string transactions = 1;`

**Review by DimaCantemir — CHANGES_REQUESTED** [review]

File: proto/api.proto (hunk 3 above)
Comment (Lanikean) [reply proto/api.proto]: you requested a response to the user like 200 OK

**Review by DimaCantemir — APPROVED** [review]

**Pull request merged**